taptargetview/src/main/java/com/getkeepsafe/taptargetview/TapTargetView.java

```diff
@@ -959,7 +1153,25 @@ int getTotalTextHeight() {
       return titleLayout.getHeight() + TEXT_SPACING;
     }
 
-    return titleLayout.getHeight() + descriptionLayout.getHeight() + TEXT_SPACING;
+    if (target.sequenceTargetCount == 0) {
+      return titleLayout.getHeight() + descriptionLayout.getHeight() + TEXT_SPACING;
+    }
+
+
+    int sequenceMaxHeight = 0;
+    if (target.sequenceCurrentTargetIndex < target.sequenceTargetCount) {
+      if (skipLayout != null && nextLayout != null) {
+        sequenceMaxHeight = Math.max(skipLayout.getHeight(), nextLayout.getHeight());
+      } else if (skipLayout != null) {
+        sequenceMaxHeight = skipLayout.getHeight();
+      } else {
+        sequenceMaxHeight = nextLayout.getHeight();
```

Comment:
Here same problem when both boolean variables are null

Reply:
Yeah, I used the same logic hahah